core/src/main/java/com/adobe/aio/util/feign/FeignUtil.java

```diff
@@ -41,6 +44,7 @@ public static Feign.Builder getBaseBuilder() {
         .logLevel(Level.NONE)
         //.logLevel(Level.FULL) // use this instead when debugging
         .decode404()
+        .retryer(new Retryer.Default(1000, SECONDS.toMillis(4), 3))
```

Comment:
don't you think we should allow the users to specify the retries through conf here ?
same as for the log level see https://github.com/adobe/aio-lib-java/issues/146

Reply:
ok let me change this
I'll add also the log level

Reply:
done :)